malaria24/ona/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+from .models import InboundSMS
+
+
+class InboundSMSSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = InboundSMS
+        exclude = ('created_at',)
```

Comment:
I don't really have a strong opinion about this, but I'm wondering if we should use the timestamp given to us or not?

If we do, it will allow us to import older messages through the API, and have accurate timestamps for message that got stuck in a queue somewhere. But then we lose the actual time that the request was made to this API. Just wanted to throw it out there, you can decide what you think is best.